dowhy/causal_refuters/data_subset_refuter.py

```diff
@@ -19,7 +19,7 @@ class DataSubsetRefuter(CausalRefuter):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._subset_fraction = kwargs.pop("subset_fraction", 0.8)
-        self._num_simulations = kwargs.pop("num_simulations", 200)
+        self._num_simulations = kwargs.pop("num_simulations", 30)
```

Comment:
shouldn't this be 100?
Actually, a better way of managing this is to define a constant member in the base refuter class, and then have each inherited class use that constant. That way, we can easily change the default without having to change each refuter.

Reply:
Added a constant in CausalRefuter.